viewing_party/party.py

```diff
@@ -0,0 +1,255 @@
+#Wave 1
+def create_movie(title,genre,rating):
+    if title and genre and rating:
+        movie_dict = {"title": title, 
+        "genre": genre,
+         "rating": rating,
+        }
+        return movie_dict
+    else:
+        return None
```

Comment:
👍 

Another way of writing this function is to use a 'guard clause'. A guard clause checks for a failed case early in a function.

```suggestion
    if not title or not genre or not rating:
        return None
        
    movie = {}
    movie_dict["title"] = title
    movie_dict["genre"] = genre
    movie_dict["rating"] = rating
    return movie_dict
```

Notice how this unindents the logic to create a movie and it is now more of a focus for the function. This is a common pattern used in Go. C, and other languages as well. 

We can further refactor this function by building the dictionary directly in the return statement:

```suggestion
    if not title or not genre or not rating:
        return None
        
    return {
        "title": title,
        "genre": genre,
        "rating": rating
    }
```